ouroboros-network/src/Ouroboros/Network/PeerSelection/Churn.hs

```diff
@@ -77,79 +114,59 @@ peerChurnGovernor tracer deadlineChurnInterval bulkChurnInterval psOverallTimeou
         writeTVar churnModeVar mode
         return mode
 
-    -- TODO: #3396 revisit the policy for genesis
-    increaseActivePeers :: ChurnMode -> STM m ()
-    increaseActivePeers mode =
+    increaseActivePeers :: ChurnRegime -> PeerSelectionTargets -> STM m ()
+    increaseActivePeers mode base =
```

Comment:
You should more consistently use the name `regime` rather than `mode` (there are too many modes around to get confused when reading the code).